docs/content/Schema/Reference/segments.md

```diff
@@ -12,7 +12,7 @@ redirect_from:
 
 Segments are predefined filters. You can use segments to define complex
```

Comment:
Not really sure how to make this page consistent, as it only has one parameter: `sql`.

Also not sure what to do about the "Segments vs Dimension Filters" section, as it isn't really reference documentation. Could/should we move it to its' own recipe perhaps, and then keep this file purely as reference docs for segments?